src/package.ts

```diff
@@ -0,0 +1,57 @@
+import { privateEncrypt } from 'crypto';
+import contextKeys from './contextKeys';
+import { exec, pathExists } from './utilities';
+
+export class SPMPackage {
```

Comment:
Can we just name this `Package`?

I assume there's no confusion here as to what kind of package we're talking about? If there is, I'd prefer a simpler `SwiftPackage` over `SPMPackage` (SPM is not the official abbreviation) or `SwiftPMPackage` (which has repeated information, as the P also stands for Package).

Reply:
I chose `SPMPackage` because `Package` is too close to `package` which is a keyword in Typescript. Can still change if you feel it is necessary.